algorithms/ar-hsrt/csharp/HeapSort.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+
+public class HeapSort
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6357874)